catalog/pages/adaptive_modal/index.md

```diff
@@ -30,6 +30,9 @@ rows:
   - Prop: displayTop
     Type: bool
     Notes: Optional. Default false. Indicates, if the modal should open at the top of the page or in the center (default)
+  - Prop: fullscreen

```

Comment:
Can you change the prop name to isFullscreen for consistency? Thank you.

Reply:
Done

Reply:
Thanks! 😄